packages/tfm-ror51-rubygem-railties/tfm-ror51-rubygem-railties.spec

```diff
@@ -0,0 +1,75 @@
+# Generated from railties-5.1.4.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name railties
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 5.1.4
+Release: 1%{?dist}
+Summary: Tools for creating, working with, and running Rails applications
+Group:   Development/Languages
+License: MIT
+URL:     http://rubyonrails.org
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby >= 2.2.2
+BuildRequires: %{?scl_prefix}rubygem(actionview) = 5.1.4
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%{?scl:Requires: %{?scl_prefix}runtime}
+
+%description
+Rails internals: application bootup, plugins, generators, and rake tasks.
+
+
+%package doc
+Summary:   Documentation for %{name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+
+mkdir -p %{buildroot}%{_bindir}
+cp -pa .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
```

Comment:
This fails because there are no bins. I suspect they're in ```exe```
```
+ xargs chmod a+x
+ xargs chmod a+x
find: '/builddir/build/BUILDROOT/tfm-ror51-rubygem-railties-5.1.4-1.el7.centos.x86_64/opt/rh/tfm-ror51/root/usr/share/gems/gems/railties-5.1.4/bin': No such file or directory
find: '/builddir/build/BUILDROOT/tfm-ror51-rubygem-railties-5.1.4-1.el7.centos.x86_64/opt/rh/tfm-ror51/root/usr/share/gems/gems/railties-5.1.4/bin': No such file or directory
chmod: missing operand after 'a+x'
chmod: missing operand after 'a+x'
```